src/main/kotlin/world/anhgelus/msmp/basicmazeworldgenerator/api/WinningHandler.kt

```diff
@@ -46,9 +64,13 @@ abstract class WinningHandler {
         ChatHelper.sendSuccess("And the winners are:")
         winners.forEach {
```

Comment:
Switch getOnlinePlayer and winner loop 
Because you send Win at first and instantly send you Lose

Reply:
Fixed